SilKit/source/util/StringHelpers.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2024 Vector Informatik GmbH
```

Comment:
Hier braucht es noch den include guard:

```suggestion
#pragma once

// Copyright (c) 2024 Vector Informatik GmbH
```